File: Python/uberX.py
'''
Created Date: Friday July 30th 2021 6:44:49 pm
Author: Andrés X. Vargas
-----
Last Modified: Friday July 30th 2021 11:27:26 pm
Modified By: the developer known as Andrés X. Vargas at <axvargas@fiec.espol.edu.ec>
-----
Copyright (c) 2021 MattuApps
'''

from car import Car

class UberX(Car):
    brand = str
    model = str

    def __init__(self, license, driver, brand, model):
        super().__init__(license, driver)
        self.brand = brand
        self.model = model

File: JS/UberX.js
/*
 * Created Date: Friday July 30th 2021 11:47:12 pm
 * Author: Andrés X. Vargas
 * -----
 * Last Modified: Friday July 30th 2021 11:49:25 pm
 * Modified By: the developer known as Andrés X. Vargas at <axvargas@fiec.espol.edu.ec>
 * -----
 * Copyright (c) 2021 MattuApps
 */
class UberX extends Car{
    constructor(license, driver, brand, model) {
        super(license, driver)
        this.brand = brand
        this.model = model
    }
}